flickit-assessment-data/src/main/java/org/flickit/assessment/data/jpa/kit/kitversion/KitVersionJpaRepository.java

```diff
@@ -17,8 +17,11 @@ public interface KitVersionJpaRepository extends JpaRepository<KitVersionJpaEnti
     @Modifying
     @Query("""
             UPDATE KitVersionJpaEntity k
-            SET k.status = :status
+            SET k.status = :status,
+                k.statusVersion = :statusVersion
             WHERE k.id = :id
```

Comment:
**Suggestion:** Ensure the SQL query in the updateStatus method handles potential null values for statusVersion to avoid unintended behavior or errors. [general, importance: 7]
```suggestion
            UPDATE KitVersionJpaEntity k
            SET k.status = :status,
                k.statusVersion = COALESCE(:statusVersion, k.statusVersion)
            WHERE k.id = :id
```